code/ui/blocks/src/blocks/external/ExternalPreview.ts

```diff
@@ -36,7 +36,7 @@ export class ExternalPreview<TRenderer extends Renderer = Renderer> extends Prev
   private moduleExportsByImportPath: Record<Path, ModuleExports> = {};
 
   constructor(public projectAnnotations: ProjectAnnotations<TRenderer>) {
-    super(new Channel());
+    super(new Channel({}));
```

Comment:
is this change needed? seems like the constructor has a default value if not passed? I'm asking because if it is needed, then this seems technically a breaking change

Reply:
runtime? no.
I made the type stricter though.

Creating a Channel without a transport is completely pointless, who creates channels other then us in the core?

Reply:
got it, and I don't know, so I thought I'd flag this, on the safe side. Feel free to ignore this then!